projects/kit/components/input-slider/input-slider.style.less

```diff
@@ -0,0 +1,34 @@
+// TODO: use tui-textfield:has([tuiSlider])
+tui-textfield._hide-buttons .t-input-number-buttons {
+    display: none !important;
+}
+
+tui-textfield._hide-buttons {
+    --t-input-number-offset-end: 0px;
+}
+
+tui-textfield._hide-buttons [tuiInputNumber]._with-buttons {
+    border-radius: inherit;
+}
```

Comment:
Oops, It is codebase from my previous iteration of new `InputSlider`.

Not relevent for the latests version of `InputSlider` 😄

Reply:
https://github.com/taiga-family/taiga-ui/pull/10288/commits/2bfb1a514c515a312dd56b489bf74e8921714b76